code/modules/reagents/Chemistry-Goon-420BlazeIt.dm

```diff
@@ -143,4 +143,365 @@ datum/reagent/crank/addiction_act_stage4(var/mob/living/M as mob)
+	M.hallucination += 10
+	if(M.canmove && !istype(M.loc, /turf/space))
+		step(M, pick(cardinal))
+		step(M, pick(cardinal))
+	..()
+	return
+
+/datum/chemical_reaction/bath_salts
+	name = "bath_salts"
+	id = "bath_salts"
+	result = "bath_salts"
+	required_reagents = list("????" = 1, "saltpetre" = 1, "nutriment" = 1, "cleaner" = 1, "enzyme" = 1, "tea" = 1, "mercury" = 1)
+	result_amount = 7
+	required_temp = 374
+
+/datum/reagent/bath_salts/overdose_process(var/mob/living/M as mob)
+	M.hallucination += 10
+	if(M.canmove && !istype(M.loc, /turf/space))
+		for(var/i = 0, i < 8, i++)
+			step(M, pick(cardinal))
+	if(prob(20))
+		M.emote(pick("twitch","drool","moan"))
+	if(prob(33))
+		var/obj/item/I = M.get_active_hand()
+		if(I)
+			M.drop_item()
+	..()
+	return
+
+/datum/reagent/bath_salts/addiction_act_stage1(var/mob/living/M as mob)
+	M.hallucination += 10
+	if(M.canmove && !istype(M.loc, /turf/space))
+		for(var/i = 0, i < 8, i++)
+			step(M, pick(cardinal))
+	M.Jitter(5)
+	M.adjustBrainLoss(10)
+	if(prob(20))
+		M.emote(pick("twitch","drool","moan"))
+	..()
+	return
+/datum/reagent/bath_salts/addiction_act_stage2(var/mob/living/M as mob)
+	M.hallucination += 20
+	if(M.canmove && !istype(M.loc, /turf/space))
+		for(var/i = 0, i < 8, i++)
+			step(M, pick(cardinal))
+	M.Jitter(10)
+	M.Dizzy(10)
+	M.adjustBrainLoss(10)
+	if(prob(30))
+		M.emote(pick("twitch","drool","moan"))
+	..()
+	return
+/datum/reagent/bath_salts/addiction_act_stage3(var/mob/living/M as mob)
+	M.hallucination += 30
+	if(M.canmove && !istype(M.loc, /turf/space))
+		for(var/i = 0, i < 12, i++)
+			step(M, pick(cardinal))
+	M.Jitter(15)
+	M.Dizzy(15)
+	M.adjustBrainLoss(10)
+	if(prob(40))
+		M.emote(pick("twitch","drool","moan"))
+	..()
+	return
+/datum/reagent/bath_salts/addiction_act_stage4(var/mob/living/carbon/human/M as mob)
+	M.hallucination += 40
+	if(M.canmove && !istype(M.loc, /turf/space))
+		for(var/i = 0, i < 16, i++)
+			step(M, pick(cardinal))
+	M.Jitter(50)
+	M.Dizzy(50)
+	M.adjustToxLoss(5)
+	M.adjustBrainLoss(10)
+	if(prob(50))
+		M.emote(pick("twitch","drool","moan"))
+	..()
+	return
+
+/datum/chemical_reaction/aranesp
+	name = "aranesp"
+	id = "aranesp"
+	result = "aranesp"
+	required_reagents = list("epinephrine" = 1, "atropine" = 1, "morphine" = 1)
+	result_amount = 3
+
+/datum/reagent/aranesp
+	name = "Aranesp"
+	id = "aranesp"
+	description = "Volatile."
+	reagent_state = LIQUID
+	color = "#60A584" // rgb: 96, 165, 132
+
+/datum/reagent/aranesp/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	var/high_message = pick("You feel amped up.", "You feel ready.", "You feel like you can push it to the limit.")
+	if(prob(5))
+		M << "<span class='notice'>[high_message]</span>"
+	M.adjustStaminaLoss(-35)
+	M.adjustToxLoss(1)
+	if(prob(rand(1,100)))
+		M.losebreath++
+		M.adjustOxyLoss(20)
+	..()
+	return
+
+/obj/item/weapon/reagent_containers/food/drinks/muriatic_acid
+	name = "jug of muriatic acid"
+	desc = "Fuck me, we needed those cooks."
```

Comment:
Check out the small, niche show with a cult following called "Breaking Bad". You can easily miss the references because it was relatively unknown for the mainstream public and only a single channel aired it. Received mixed reactions from critics but fans claim " its the new firefly." Ran for the relatively short period of just 5 and a half season, got axed because low viewer ratings. Hope that explains it.